src/app/store/technical-records/reducers/technical-record-service.reducer.ts

```diff
@@ -33,6 +40,11 @@ export const initialState: TechnicalRecordServiceState = {
   loading: false
 };
 
+export const initialState2: TechnicalRecordServiceState = {
```

Comment:
Seems this `initialState2` is only being used in tests, would make more sense to have in the `spec` file(s). 

Also unsure as to why it's needed, if a test needs some data, it would be trivial to provide some mock data for that test on a case by case basis, by spreading `initialState` and adding some records if the test needs it. 

If multiple tests need the same initial data, it could go in a `beforeEach` for example

Reply:
Yeah I think this needs binning, I don't think it's used here....

Reply:
Idk why I put this here, I was very tired, sorry :(